app/src/lib/editors/darwin.ts

```diff
@@ -144,6 +144,10 @@ const editors: IDarwinExternalEditor[] = [
     name: 'Lite XL',
     bundleIdentifiers: ['com.lite-xl'],
   },
+  {
+    name: 'JetBrains Fleet',
```

Comment:
Other JetBrains IDEs on this list do not have the JetBrains prefix, neither should Fleet. This is only on Mac though, for some reason the JetBrains prefix has been normalized on Windows, apparently.

Reply:
You're absolutely right 🙌